be/src/exec/vectorized/olap_scan_node.cpp

```diff
@@ -553,6 +574,48 @@ Status OlapScanNode::_capture_tablet_rowsets() {
     return Status::OK();
 }
 
+struct TypeMemoryUsed {
+    template <FieldType type>
+    size_t operator()() {
+        switch (type) {
+        case OLAP_FIELD_TYPE_VARCHAR:
+        case OLAP_FIELD_TYPE_CHAR:
+        case OLAP_FIELD_TYPE_JSON:
+            return 100;
+#define M(TYPE) return 100;
+            APPLY_FOR_COMPLEX_OLAP_FIELD_TYPE(M)
+#undef M
+        default:
+            return 0;
+        }
+    }
+};
+
+size_t OlapScanNode::_scanner_concurrency() {
```

Comment:
The main memory consumption of Scanner has two parts, each accounting for 1/2, one is Chunk and the other is Page. Chunk is generally obtained from ColumnPool and will not be calculated into the query memory, it's hard to calculated  based on Page, so it's also ok based on Chunk.

Reply:
OK